packages/redux/src/store.ts

```diff
@@ -37,6 +37,7 @@ export interface WorkspaceSettings {
   timeZone: string;
   showTimeZone: boolean;
   showTSeparator: boolean;
+  truncateNumbersWithPound: boolean;
```

Comment:
While this does work... and I know we talked about it a bit... but since we're writing to workspace data I want to make sure we make it flexible enough we don't have to do a data migration later. I think it would be better to save it as `truncationChar?: string;` in `defaultDecimalFormatOptions`, `defaultIntegerFormatOptions`. 
The check box in settings will need to trigger to update both, if for some reason they're out of sync then just show the intermediate checkbox state (they shouldn't get out of sync though).
Having just this option kind of boxes us in.

Reply:
Per our call today, we'll just leave it as is since there aren't any current requests to make this a per-column option. If that's the case then we'd want to spec out what other truncation styles are needed (if any) and if this setting should be changed to something else or if it can be left as the global setting and then have the model account for specific formatters based on column name.